receiver/journaldreceiver/README.md

```diff
@@ -55,6 +55,50 @@ Each operator performs a simple responsibility, such as parsing a timestamp or J
 - Only parsers and general purpose operators should be used.
 
 ### Example Configurations
+#### Minimal configuration
+
+The following configuration is the minimal configuration to read
+journald logs:
+
+```yaml
+receivers:
+  journald:
+```
+will be passed to journalctl as the following arguments: `journalctl
+... --priority info`. This will read the 10 most recent entries and
+any subsequent entry.  `--priority info` is the default priority, the
+following examples will omit it for simplicity.
+
+#### Cursor tracking
+```yaml
+receivers:
+  journald:
+  storage: storage: file_storage/journald
+
+extensions:
+  file_storage/journald:
+    directory: .
+
+service:
+  extensions: [file_storage/journald]
+```
+
+A file called `receiver_journald_` will be created in the current
```

Comment:
A file being created is a detail of the File Storage extension that I don't think should be described in Journald receiver's docs.